apps/dashboard/tailwind.config.js

```diff
@@ -153,15 +153,15 @@ export default {
         },
         swing: {
           '0%, 9.9%, 100%': { transform: 'rotate(0deg)' },
-          '10%': { transform: 'rotate(2deg)' },
-          '20%': { transform: 'rotate(-2deg)' },
-          '30%': { transform: 'rotate(1.5deg)' },
-          '40%': { transform: 'rotate(-1.5deg)' },
-          '50%': { transform: 'rotate(1deg)' },
-          '60%': { transform: 'rotate(-1deg)' },
-          '70%': { transform: 'rotate(0.5deg)' },
-          '80%': { transform: 'rotate(-0.5deg)' },
-          '90%': { transform: 'rotate(0.2deg)' },
+          '10%': { transform: 'rotate(3deg)' },
+          '20%': { transform: 'rotate(-3deg)' },
+          '30%': { transform: 'rotate(2.25deg)' },
+          '40%': { transform: 'rotate(-2.25deg)' },
+          '50%': { transform: 'rotate(1.5deg)' },
+          '60%': { transform: 'rotate(-1.5deg)' },
+          '70%': { transform: 'rotate(0.75deg)' },
+          '80%': { transform: 'rotate(-0.75deg)' },
+          '90%': { transform: 'rotate(0.3deg)' },
```

Comment:
Making the jingle animation a bit stronger, it was difficult to see earlier. It's still subtle enough.